packages/ragbits-core/src/ragbits/core/vector_stores/base.py

```diff
@@ -29,9 +31,12 @@ class VectorStore(ABC):
     A class with an implementation of Vector Store, allowing to store and retrieve vectors by similarity function.
     """
 
-    def __init__(self, default_options: VectorStoreOptions | None = None) -> None:
+    metadata_store: MetadataStore | None
```

Comment:
This'll be buggy, if we create multiple instances of `VectorStore`. Let's make it a private attribute instance.